R/classicalmetaanalysis.R

```diff
@@ -15,83 +15,183 @@
+  # simple ma specific
+  "heterogeneityModelTerms", "heterogeneityModelIncludeIntercept", "heterogeneityModelLink",
+  "permutationTest", "permutationTestIteration", "permutationTestType", "setSeed", "seed",
+  # multilevel/multivariate specific
+  "randomEffects", "randomEffectsSpecification",
+  "computeCovarianceMatrix", "computeCovarianceMatrix"
+)
+.maForestPlotDependencies <- c(
+  .maDependencies, "transformEffectSize", "confidenceIntervalsLevel",
+  "forestPlotStudyInformation",
+  "forestPlotStudyInformationAllVariables",
+  "forestPlotStudyInformationSelectedVariables",
+  "forestPlotStudyInformationSelectedVariablesSettings",
+  "forestPlotStudyInformationPredictedEffects",
+  "forestPlotStudyInformationStudyWeights",
+  "forestPlotStudyInformationOrderBy",
+  "forestPlotStudyInformationOrderAscending",
+  "forestPlotEstimatedMarginalMeans",
+  "forestPlotEstimatedMarginalMeansModelVariables",
+  "forestPlotEstimatedMarginalMeansSelectedVariables",
+  "forestPlotEstimatedMarginalMeansTermTests",
+  "forestPlotEstimatedMarginalMeansCoefficientTests",
+  "forestPlotEstimatedMarginalMeansCoefficientTestsAgainst",
+  "forestPlotEstimatedMarginalMeansAdjustedEffectSizeEstimate",
+  "forestPlotModelInformation",
+  "forestPlotPooledEffectSizeEstimate",
+  "forestPlotPooledEffectSizeTest",
+  "forestPlotResidualHeterogeneityTest",
+  "forestPlotResidualHeterogeneityEstimate",
+  "forestPlotEffectSizeModerationTest",
+  "forestPlotHeterogeneityModerationTest",
+  "forestPlotPredictionIntervals",
+  "forestPlotEstimatesAndConfidenceIntervals",
+  "forestPlotTestsInRightPanel",
+  "forestPlotMappingColor",
+  "forestPlotMappingShape",
+  "forestPlotRelativeSizeEstimates",
+  "forestPlotRelativeSizeText",
+  "forestPlotRelativeSizeAxisLabels",
+  "forestPlotRelativeSizeRow",
+  "forestPlotRelativeSizeLeftPanel",
+  "forestPlotRelativeSizeMiddlePanel",
+  "forestPlotRelativeSizeRightPanel",
+  "forestPlotAuxiliaryAdjustWidthBasedOnText",
+  "forestPlotAuxiliaryDigits",
+  "forestPlotAuxiliaryTestsInformation",
+  "forestPlotAuxiliaryPlotColor",
+  "forestPlotAuxiliaryAddVerticalLine",
+  "forestPlotAuxiliaryAddVerticalLineValue",
+  "forestPlotAuxiliaryAddVerticalLine2",
+  "forestPlotAuxiliaryAddVerticalLineValue2",
+  "forestPlotAuxiliaryEffectLabel",
+  "forestPlotAuxiliarySetXAxisLimit",
+  "forestPlotAuxiliarySetXAxisLimitLower",
+  "forestPlotAuxiliarySetXAxisLimitUpper",
+  "forestPlotStudyInformationSecondaryConfidenceInterval",
+  "forestPlotStudyInformationSecondaryConfidenceIntervalLevel"
+)
+.maBubblePlotDependencies <- c(
+  .maDependencies, "transformEffectSize", "confidenceIntervalsLevel",
+  "bubblePlotSelectedVariable",
+  "bubblePlotSeparateLines",
+  "bubblePlotSeparatePlots",
+  "bubblePlotSdFactorCovariates",
+  "bubblePlotBubblesSize",
+  "bubblePlotBubblesRelativeSize",
+  "bubblePlotBubblesTransparency",
+  "bubblePlotBubblesJitter",
+  "bubblePlotCondifenceIntervals",
```

Comment:
```suggestion
  "bubblePlotConfidenceIntervals",
```
note that the qml has the same typo